commands/serverless-extra.go

```diff
@@ -59,6 +61,8 @@ project (a directory you have designated for functions development).  This can b
 	AddStringFlag(getMetadata, "env", "", "", "Path to environment file")
 	AddStringFlag(getMetadata, "include", "", "", "Functions or packages to include")
 	AddStringFlag(getMetadata, "exclude", "", "", "Functions or packages to exclude")
+	AddBoolFlag(getMetadata, "project-reader", "", false, "Test new project reader service")
+	getMetadata.Flags().MarkHidden("project-reader")
```

Comment:
I'm curious about the purpose of this flag. Is only intended for internal testing, or will we expose it to customers? Think I'm just missing some context.

Reply:
It will never be exposed to customers.    The plan is that `get-metadata` will _always_ using the new project-reader service once we've gained enough confidence to switch over.   The flag is "for testing only" while we're not yet ready to switch, and will help us gain the confidence by allowing us to compare behavior with and without the flag.   So the flag will eventually be removed without ever being exposed.